app/assets/javascripts/discourse/app/controllers/history.js

```diff
@@ -215,6 +227,15 @@ export default Controller.extend(ModalFunctionality, {
     return this.currentUser && this.currentUser.get("staff");
   },
 
+  @discourseComputed()
+  canPurge() {
+    return (
+      this.siteSettings.can_permanently_delete_revisions &&
+      this.currentUser &&
+      this.currentUser.get("staff")
```

Comment:
You should be able to do this now

```suggestion
      this.currentUser?.get("staff")
```

And you might even get away with just

```suggestion
      this.currentUser?.staff
```